src/decomposition.jl

```diff
@@ -24,6 +24,14 @@
 #     end
 #     return
 # end
+assignsolver(n::AbstractNode, f::Function) = n.master.optimizer_builder = f
+assignsolver(n::Leaf, f::Function) = n.problem.optimizer_builder = f
+function assignsolver(nodes::Vector{<:AbstractNode}, f::Function)
+    for n in nodes
+        assignsolver(n, f)
+    end
+    return
```

Comment:
should return `f` as `assignsolver(n::AbstractNode, f::Function)`

Reply:
ok